generation/other/dxc/dxcapi/generate.rsp

```diff
@@ -0,0 +1,54 @@
+@../../settings.rsp
+@../../remap.rsp
+--exclude
+DxcValidatorFlags_Default
+DxcValidatorFlags_InPlaceEdit
+DxcValidatorFlags_RootSignatureOnly
+DxcValidatorFlags_ModuleOnly
+DxcValidatorFlags_ValidMask
+DxcVersionInfoFlags_None
+DxcVersionInfoFlags_Debug
+DxcVersionInfoFlags_Internal
+CLSID_DxcCompiler
+CLSID_DxcLinker
+CLSID_DxcDiaDataSource
+CLSID_DxcLibrary
+CLSID_DxcValidator
+CLSID_DxcAssembler
+CLSID_DxcContainerReflection
+CLSID_DxcOptimizer
+CLSID_DxcContainerBuilder
+CLSID_DxcCompilerArgs
+CLSID_DxcUtils
+--file
+other-dxcapi.h
+--output
+../../../sources/Interop/Windows/um/dxcapi
```

Comment:
The output for these should be `other/dxc/dxcapi` to help clarify that they aren't "in-box" by default.
Likewise, any members already covered by `um/dxcapi` should be in the excludes list.